scripts/gen-kmem.py

```diff
@@ -0,0 +1,208 @@
+#! /usr/bin/env python3
```

Comment:
Okay, I believe you! :D
Actually, I've been convinced by studying the resulting binaries.
Maybe we can consider re-writing/updating it when transitioning to the "framework". For now, don't bother.
Using PEP-8 indentation might improve readability, though.